signingscript/test/integration/test_autograph.py

```diff
@@ -0,0 +1,224 @@
+import aiohttp
+import asyncio
+import copy
+import json
+import os
+import pytest
+import subprocess
+import shutil
+import tempfile
+import zipfile
+
+from mardor.cli import do_verify
+from scriptworker.utils import makedirs, download_file
```

Comment:
It doesn't look like `download_file` is getting used, unless I missed something.

Reply:
Fixed